notebooks/Viewing 3D volumetric data with matplotlib.ipynb

```diff
@@ -14,9 +17,13 @@
     "editable": true
    },
    "source": [
-    "Most readers are familiar with image data, taken with ordinary cameras (these are often called \"natural images\" in the scientific literature), but also with specialized instruments such as microscopes or telescopes.\n",
+    "Most of you are familiar with image data, taken with ordinary cameras (these are often called \"natural images\" in the scientific literature), but also with specialized instruments, such as microscopes or telescopes. When you're working with images in Python, you'll most often make use of the Matplotlib library, Python's 2D plotting library which produces publication quality figures in a variety of hard copy formats and interactive environments across platforms.\n",
+    "\n",
+    "Today's tutorial will show you how you can easily view 3D volumetric data with Matplotlib. \"3D volumetric data\" you say? Volumetric data is very common nowadays: it is captured by various technologies, e.g. MRI, CT, PET, USCT or echolocation. As you can already guess, this type of data plays an important role in medicine. \n",
     "\n",
-    "Using matplotlib to display such data is a straightforward call to the `imshow` function.\n",
+    "This tutorial will show you how to import and display image data, and how you can make a fully functional slice viewer to view all the slices in our MRI volume without any interference.\n",
```

Comment:
This is now quite repetitive ("today's tutorial will show you...", "this tutorial will show you..."). I'm also not a fan of using the standard matplotlib spiel to introduce it — it seems out of place to talk about plots in this context. Now that I see what you're going for, how about the following:

> Most of you are familiar with image data, taken with ordinary cameras (these are often called "natural images" in the scientific literature), but also with specialized instruments, such as microscopes or telescopes. When working with images in Python, the most common way to display them is using the `imshow` function of *Matplotlib*, Python's most popular plotting library.

> In this tutorial, we'll show you how to extend this function to display *3D volumetric data*, which you can think of as a stack of images. Together, they describe a 3D structure. For example, magnetic resonance imaging (MRI) and computed tomography (CT) scans measure the 3D structure inside the human body; X-ray microtomography measures the 3D structure inside materials such as glass, or metal alloys; and light-sheet microscopes measure fluorescent particles inside biological tissues. We'll demonstrate how to download an MRI dataset and display the slices using matplotlib. But first, let's go over some basics: how to display images with Matplotlib's `imshow`: